packages/application-components/src/index.ts

```diff
@@ -13,6 +13,7 @@ export { default as InfoModalPage } from './components/modal-pages/info-modal-pa
 export { default as FormModalPage } from './components/modal-pages/form-modal-page';
 export { default as TabularModalPage } from './components/modal-pages/tabular-modal-page';
 export { default as CustomFormModalPage } from './components/modal-pages/custom-form-modal-page';
+export { default as ModalPageTopBar } from './components/modal-pages/internals/modal-page-top-bar';
```

Comment:
Now we need to use this component from the `CustomViewShell` component but it's weird to export an `internal` component.

What do you think about it?

Reply:
Are you still ok in releasing this change?

Reply:
`2` sounds good or `1` and moving the export down to `// Utilities`

Reply:
I'm fine with any option, up to you to decide. Thanks!

Reply:
I went for the second option.

Updated here: 906dfe4150dcd267dabbec87c1c50485754a5300